pkg/antctl/raw/multicluster/init.go

```diff
@@ -29,6 +29,9 @@ import (
 const (
 	defaultToken = "default-member-token"
 
+	// We comment out these ClusterSetJoinConfig fields in the generated
+	// join config file, so they can be populated by command line options of
+	// the "join" command.
```

Comment:
Maybe we can use "antctl mc join" command ?

Reply:
Sure. Changed.